.circleci/config.yml

```diff
@@ -22,13 +22,13 @@ jobs:
           command: mkdir build && cd build && cmake ..
       - run:
           name: Build
-          command: cd build && make -j4
+          command: cd build && make -j2
```

Comment:
BTW, it probably makes sense to use `MAKEFLAGS` environment variable in CircleCI job settings to get rid of adding this everywhere.